driver/node.go

```diff
@@ -36,17 +42,20 @@ func NewNodeService(
 		volumeMountService:  volumeMountService,
 		volumeResizeService: volumeResizeService,
 		volumeStatsService:  volumeStatsService,
+		forceVolumeStaging:  os.Getenv("FORCE_STAGING_SUPPORT") == "true",
 	}
 }
 
 const encryptionPassphraseKey = "encryption-passphrase"
 
 func (s *NodeService) NodeStageVolume(ctx context.Context, req *proto.NodeStageVolumeRequest) (*proto.NodeStageVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not supported")
+	// while we dont do anything here, Swarm 23.0.1 might require this
+	return &proto.NodeStageVolumeResponse{}, nil
 }
 
 func (s *NodeService) NodeUnstageVolume(ctx context.Context, req *proto.NodeUnstageVolumeRequest) (*proto.NodeUnstageVolumeResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not supported")
+	// while we dont do anything here, Swarm 23.0.1 might require this
+	return &proto.NodeUnstageVolumeResponse{}, nil
 }
```

Comment:
I would avoid implementing a hack for a bug in Moby/SwarmKit; https://github.com/moby/swarmkit/pull/3116 will address this.

Reply:
@neersighted assuming the PR will take some time to be merged / a release will take a while, in this case and if @apricote is fine with it, I would like to still merge it tbh. This way we can get some real world experience with this driver even though it is not perfect yet. The capabilities are hidden behind a featureflag that does not impact anyone besides new Swarm users.

Once the PR is merged, I will happily create another PR to remove the hack. I totally understand if we don't want to go ahead with this though, as this is not my decision to make :)